ouroboros-consensus/.dir-locals.el

```diff
@@ -1,5 +0,0 @@
-;;; Directory Local Variables
```

Comment:
We might want to consider using [editor config](https://editorconfig.org/) instead. Eg https://github.com/input-output-hk/decentralized-software-updates/blob/master/.editorconfig